docs/release-notes.md

```diff
@@ -4,6 +4,42 @@ nav_order: 9
 
 # Release notes
 
+## Butane 0.15.0 (2022-06-23)
+
+### Breaking changes
+
+- Return selected `compression` field value from `MakeDataURL()` _(Go API)_
+
+### Features
+
+- Add Flatcar support 1.0.0 and 1.1.0-exp, targeting Ignition spec 3.30 and 3.40-exp respectively
+- Stabilize OpenShift spec 4.11.0, targeting Ignition spec 3.2.0
+- Add arm64 support to container
+- Add GRUB password support _(fcos 1.5.0-exp, openshift 4.12.0-exp)_
+- Add `AddFromCommonObject()` `TranslationSet.Map()` _(Go API)_
+
+### Bug fixes
+
+- Fix `build` script on Windows
+- Fix local file inclusion on Windows
+
+### Misc. changes
+
+- Derive container from Fedora image to support use in multi-stage builds
+- Fail if setuid/setgid/sticky mode bits specified _(fcos 1.0.0 - 1.4.0, openshift 4.8.0 - 4.11.0-exp )_
```

Comment:
We're stabilizing 4.11.0, so it's not `-exp` anymore.

And this doesn't affect FCOS.

Reply:
Also, this only affects `openshift` 4.10.0+.